parm/config/gfs/config.wave

```diff
@@ -25,26 +25,60 @@ export CDUMPwave="${RUN}wave"
 export CDUMPRSTwave="gdas"
 
 # Grids for wave model
-# GFSv16
-export waveGRD=${waveGRD:-'gnh_10m aoc_9km gsh_15m'}
-export waveGRDN=${waveGRDN:-'1 2 3'} # gridnumber for ww3_multi
-export waveGRDG=${waveGRDG:-'10 20 30'} # gridgroup for ww3_multi
-export USE_WAV_RMP=${USE_WAV_RMP:-'YES'} #yes/no rmp grid remapping pre-processed coefficients
-export waveMULTIGRID=${waveMULTIGRID:-'.true.'}
-export MESH_WAV=${MESH_WAV:-'mesh.gwes_30m.nc'}
-
-# ESMF input grid
-export waveesmfGRD=${waveesmfGRD:-'glox_10m'} # input grid
+export waveGRD=${waveGRD:-'mx025'}
+
+#Grid dependent variables for various grids
+case "${waveGRD}" in
+  "gnh_10m;aoc_9km;gsh_15m")
+    #GFSv16 settings: 
+    waveGRDN='1 2 3'
+    waveGRDG='10 20 30'
+    USE_WAV_RMP='YES'
+    waveMULTIGRID='.true.'
+    IOSRV='3' 
+    MESH_WAV=' '
+    waveesmfGRD='glox_10m'
+    waveuoutpGRD='points'
+    waveinterpGRD='glo_15mxt at_10m ep_10m wc_10m glo_30m'
+    wavepostGRD='gnh_10m aoc_9km gsh_15m'
+    ;;
+  "gwes_30m")
+    #Grid used for P8
+    waveinterpGRD=' '
+    wavepostGRD='gwes_30m'
+    ;;
+  "mx025")
+    #Grid used for HR1 (tripolar 1/4 deg)
+    waveinterpGRD='reg025'
+    wavepostGRD=' '
+    ;;
+  "glo_025") 
+    #GEFSv13 regular lat/lon 1/4 deg grid 
+    waveGRD='glo_025'
+    waveinterpGRD=' '
+    wavepostGRD='glo_025'
+    ;;
+esac
+
+
+export waveGRD=${waveGRD:-'mx025'}
+export waveGRDN=${waveGRDN:-'1'} # gridnumber for ww3_multi
+export waveGRDG=${waveGRDG:-'10'} # gridgroup for ww3_multi
+export USE_WAV_RMP=${USE_WAV_RMP:-'NO'} #yes/no rmp grid remapping pre-processed coefficients
+export waveMULTIGRID=${waveMULTIGRID:-'.false.'} #true/false for multi or shel 
+export MESH_WAV=${MESH_WAV:-"mesh.${waveGRD}.nc"}  #Mesh grid for wave model for CMEPS
```

Comment:
Probably want to move these before the `case` block and set as defaults.

Reply:
So similar to above comments, my idea was the set any grid dependent value in the case block above, and then default to the default values otherwise.  Is this a bad strategy?